### PR TITLE
Bookstore: Add styling

### PR DESCRIPTION
**Pull request by ernestmusong**
**Followed design guidelines.**
However, some assets are not downloadable such as the progress bar. As a result, I have added a custom progress bar

### CODE REVIEW

**Review by OyePriscilla — COMMENTED** [review]
Hi @ernestmusong 👍, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

**You have submitted a project, that has one of the README file headers (**Project Presentation**) with empty content. Please kindly fix your live demo section that's leading to your repo; you can add a working link or you remove it if there is no live deployment at the moment. 👍 This will make your repo more engaging. ℹ️**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/styles/App.css [hunk 1]
```diff
@@ -17,140 +22,188 @@
...
 main {
-  padding-left: 50px;
-  padding-right: 50px;
-  padding-top: 50px;
-  width: 100%;
+  margin-left: 6.25rem;
+  margin-right: 6.25rem;
+  padding-top: 2.375rem;
+  padding-left: 6.25rem;
+  padding-right: 6.25rem;
```
Comment (BertrandConxy) [inline src/styles/App.css:99]: - [ ] Good job so far 👏 👏 In addition to what you have done, I would suggest that you use the Circular progress bar in order to match the design given. Here is the comparison 👇 
Given design
image

Your design
image

If you don't want to build it yourself, you can use a package and there are a lot of such packages out there.
One of them I would suggest is [this one here](https://www.npmjs.com/package/react-circular-progressbar)

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @ernestmusong, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/App.css (hunk 1 above)
Comment (ernestmusong) [reply src/styles/App.css:99]: > * [ ]  Good job so far 👏 👏 In addition to what you have done, I would suggest that you use the Circular progress bar in order to match the design given. Here is the comparison 👇
>   Given design
>   image
> 
> Your design image
> 
> If you don't want to build it yourself, you can use a package and there are a lot of such packages out there. One of them I would suggest is [this one here](https://www.npmjs.com/package/react-circular-progressbar)

Thanks @BertrandConxy for your providing me link to the npm circular svg package. I have no added the circular progress bar.

**Review by fobadara — APPROVED** [review]
Hi @ernestmusong, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
✅ Implemented reviewer's suggestions accurately

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**